uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Menu/Navigation.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+interface CRM_HRCore_Menu_Navigation {
```

Comment:
Will the interface ever possibly have more than one concrete class?

Reply:
Don't think so. Its not necessary right?

Reply:
Well I'm not sure about how implementation will be in the future. Maybe you'll have some instance of it for sub menus or something. I'm fine with having an interface, just was curious